fedora/templates/csv/packages_installed.csv

```diff
@@ -3,6 +3,7 @@ audit
 chrony
 cronie
 dconf
+dracut-fips
```

Comment:
This should not be enabled here. Please remove this. All changes for dracut-fips should be made here. `linux_os/guide/system/software/integrity/fips/package_dracut-fips_installed`

Reply:
@redhatrises fixed